examples/research_projects/fsner/README.md

```diff
@@ -0,0 +1,86 @@
+# FSNER
+
+Implemented by [sayef](https://huggingface.co/sayef).
+
+## Overview
+
+The FSNER model was proposed in [Example-Based Named Entity Recognition](https://arxiv.org/abs/2008.10570) by Morteza Ziyadi, Yuting Sun, Abhishek Goswami, Jade Huang, Weizhu Chen. To identify entity spans in a new domain, it uses a train-free few-shot learning approach inspired by question-answering.
+
+
+
+## Abstract
+----
+> We present a novel approach to named entity recognition (NER) in the presence of scarce data that we call example-based NER. Our train-free few-shot learning approach takes inspiration from question-answering to identify entity spans in a new and unseen domain. In comparison with the current state-of-the-art, the proposed method performs significantly better, especially when using a low number of support examples.
+
+
+
+## Model Training Details
+-----
+
+| identifier        | epochs           | datasets  |
+| ---------- |:----------:| :-----:|
+| sayef/fsner-bert-base-uncased      | 10 | ontonotes5, conll2003, wnut2017, and fin (Alvarado et al.). |
+
+
+## Installation and Example Usage
+------
+
+You can use the FSNER model in two ways-
+
+1. Install as a package: `python setup.py install` and import as following code example -
+
+    or
+
+2. Change directory to `src` and import as following code example -
+
+
+
+```python
+from fsner import FSNERModel, FSNERTokenizerWrapper
+
+model = FSNERModel("sayef/fsner-bert-base-uncased")
+
+tokenizer = FSNERTokenizerUtils("sayef/fsner-bert-base-uncased")
```

Comment:
I see you're importing `FSNERTokenizerWrapper`, but then using `FSNERTokenizerUtils`.

Reply:
Mistake. Fixed in my latest [commit](https://github.com/huggingface/transformers/pull/13155/commits/bffa29558ed181a057a04db74f426f615219b4c5).